docs/02-quickstart/index.mdx

```diff
@@ -58,7 +62,13 @@ curl -X GET \
 
 With this, you're ready to start building!
 
-### 3. Start Building {#start-building}
+### 2. Start Building {#start-building}
+
+:::info
+
+Looking to use a SDK? Check out [Livepeer.js](https://livepeerjs.org/)!
+
+:::
```

Comment:
I don't actually know this `:::` syntax, but just wanna point out that the 2 new ones are inconsistent: the other one has `note` on the closing tag, while this one only has the `:::`. Which should be fixed?

Reply:
The `:::` syntax let us use [admonitions](https://docusaurus.io/docs/markdown-features/admonitions) so we can add notes or caution messages.

I was thinking of using note for prerequisite assumptions and info for more information related to the current content